_includes/head.html

```diff
@@ -9,4 +9,10 @@
     <link rel="stylesheet" href="{{site.style_baseurl}}/css/main.css">
     <!-- syntax highlighting CSS -->
     <link rel="stylesheet" href="{{ site.style_baseurl }}/css/syntax.css" />
+    <script type="text/javascript" src="js/carousel.js"></script>
```

Comment:
Add `{{ site.style_baseurl }}` to the beginning?

Reply:
Isn't that for "styles", i.e. CSS?
What about using {{ site.baseurl }}?
Or would it make sense to add a `site.js_basurl` to the config file`?

Moreover `baseurl` and `style_basurl` are bot set to `''`adding some value there might also make sense, wouldn't it?

Reply:
Hm. Probably adding a `/` at the beginning would be enough; this would match the CSS styles (The baseurl is empty, but it still adds the `/`).

Reply:
If it ain't broke? 🤷